daemon/inertia/project/repository_test.go

```diff
@@ -80,15 +79,8 @@ func TestUpdateRepository(t *testing.T) {
 }
 
 func TestCompareRemotes(t *testing.T) {
-	cwd, err := os.Getwd()
-	assert.Nil(t, err)
-
-	// Traverse back down to base directory
-	inertia := path.Dir(cwd)
-	inertia = path.Dir(inertia)
-	inertia = path.Dir(inertia)
-
-	repo, err := git.PlainOpen(inertia)
+	// Traverse back down to root directory of repository
+	repo, err := git.PlainOpen("../../../")
```

Comment:
Are we safe to assume that the repo will always be 3 levels deep from the cwd?

Reply:
Yep! `cwd` for tests is always the location of the test itself, and until the next restructure (hopefully not too soon again 😅 ) `daemon/inertia/project` will always be `daemon/inertia/project` 🎉

Reply:
Ok, I suppose the location shouldn't change too often.